images/node/Dockerfile

```diff
@@ -17,6 +17,7 @@ ADD https://copr.fedoraproject.org/coprs/maxamillion/origin-next/repo/epel-7/max
 RUN yum install -y libmnl libnetfilter_conntrack openvswitch \
     libnfnetlink iptables iproute bridge-utils procps-ng ethtool socat openssl \
     binutils xz kmod-libs kmod sysvinit-tools device-mapper-libs dbus \
+    e2fsprogs xfsprogs ceph-common glusterfs-fuse iscsi-initiator-utils nfs-utils \
```

Comment:
thank you @sdodson I just looked at that BZ. But the log messages aren't showing an error that I can tell, so i'm not 100% what we need.  Especially since gluster is the one case I think even the *progs aren't needed.